tests/test_connector.py

```diff
@@ -39,31 +39,36 @@
 from neon_mq_connector.config import Configuration
 from neon_mq_connector.connector import MQConnector, ConsumerThread
 from neon_mq_connector.utils import RepeatingTimer
+from neon_mq_connector.utils.rabbit_utils import create_mq_callback
 
 
 class MQConnectorChild(MQConnector):
 
-    def callback_func_1(self, channel, method, properties, body):
```

Comment:
Consider duplicating these tests to leave the old ones and add these changes as another set of tests